conf/routes

```diff
@@ -17,6 +17,7 @@ GET     /api2/atom/:id                  controllers.Api2.getAtom(id)
 PUT     /api2/atom/:id/asset            controllers.Api2.addAsset(id)
 
 GET     /api/youtube/categories         controllers.YoutubeCategory.list()
+PUT     /api/youtube/videos/:id/update  controllers.YoutubeVideos.update(id)
```

Comment:
Then I'll move this in the `Api2.scala` controller, because this is about updating the metadata for all platforms, it makes no sense to sit in a file called `YoutubeVideo.scala`.